dotnet/src/MicroFocus.CafApi.QueryBuilder.Vqll.Builder.SystemJson/SystemJsonFilterWriter.cs

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright 2022 Micro Focus or one of its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+using System.Text.Json;
+using MicroFocus.CafApi.QueryBuilder.Vqll.Builders.Internal;
+
+namespace MicroFocus.CafApi.QueryBuilder.Vqll.Builders.SystemJson
+{
+    public static class SystemJsonFilterWriter
+    {
+        public static void WriteToJsonArray(
+            Filter<string> filter,
+            Utf8JsonWriter jsonWriter
+        )
+        {
+            IJsonBuilder jsonBuilder = new SystemJsonBuilder(jsonWriter);
+            FilterWriter.WriteToJsonArray(filter, jsonBuilder);
+        }
+    }
```

Comment:
.NET has [Extension Methods](https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/extension-methods), which is a great feature that Java is missing.

Using it means deviating a bit from the Java implementation but it's worth it in this case.

- Rename the class (and the file) to `FilterExtension`
- Put `this` before the first parameter and rename the parameter to `that`

```suggestion
    public static class FilterExtension
    {
        public static void WriteToJsonArray(
            this Filter<string> that,
            Utf8JsonWriter jsonWriter
        )
        {
            IJsonBuilder jsonBuilder = new SystemJsonBuilder(jsonWriter);
            FilterWriter.WriteToJsonArray(that, jsonBuilder);
        }
    }
```